ci-operator/step-registry/azure/provision/service-principal/hypershift/azure-provision-service-principal-hypershift-commands.sh

```diff
@@ -40,6 +46,12 @@ for component in $COMPONENTS; do
     component_to_cert_name+=(["$component"]="$name")
 done
 
+# TODO: Remove this once the we used the automated role assignment by "--assign-service-principal-role"
+az role assignment create \
```

Comment:
I guess this command will create a "assignment" resource on `BASE_DOMAIN_RESOURCE_GROUP`, later we need to remove it in deprovision step. So here need to record the assignment id, then call `azure-deprovision-role-assignment` in deprovision chain to clean it up.

You can reference https://github.com/openshift/release/blob/db4c0b8928e126945c48318e1c94d20cda064632/ci-operator/step-registry/azure/provision/bastionhost/managed-identity/azure-provision-bastionhost-managed-identity-commands.sh#L97 to do similar things.

Reply:
Good to know, thank you! Fixed.